libcloudforensics/providers/gcp/internal/storage.py

```diff
@@ -131,6 +131,21 @@ def GetBucketACLs(self,
         ret[item['role']].append(member)
     return ret
 
+  def ListBuckets(self) -> List[Dict[str, Any]]:
+    """List buckets in a Google Cloud project.
+
+    Args:
+      None
+
+    Returns:
+      List of Object Dicts
```

Comment:
Missing return type: `List[Dict[str, Any]]: List of object dicts`. Also, maybe add a short description of what these 'object dicts' contain/represent?

Reply:
Description is on the next line :)
Return type done.

Reply:
Ah, skipped the link, sorry :) 
Thanks!